e2e/pages/nav-page.ts

```diff
@@ -1,11 +1,13 @@
 import { Page, Locator } from 'playwright';
 import { K8sPage } from './k8s-page';
 import { PortForwardPage } from './portforward-page';
+import { WSLIntegrationsPage } from './wsl-integrations-page';
 import { ImagesPage } from './images-page';
 import { TroubleshootingPage } from './troubleshooting-page';
 
 const pageConstructors = {
   K8s:             (page: Page) => new K8sPage(page),
+  WSLIntegrations:    (page: Page) => new WSLIntegrationsPage(page),
```

Comment:
Nit: alignment is off.
```suggestion
  WSLIntegrations: (page: Page) => new WSLIntegrationsPage(page),
```